### ISSUE TITLE
Copy content of directory to target

### ISSUE THREAD

**Issue by hediet** [pk-nb/sync-folders#2]
Right now, when calling `sync-folders src target`, the tool synchronizes the content from `src` to `target/src`. In my case I need to control the target directory name, but I cannot change the source directory name.
It would be great, if, with an additional flag, your tool could synchronize the content from `src` to `target`, so that a file `src/index.js` will end up in `target/index.js` (and not `target/src/index.js`).

Thanks ;) Btw. your tool is cool, I tried robocopy first, but it does not support a watch mode. Sadly, it's unusable for me due to this problem.

**Comment by aamakerlsa** [comment]
@hediet - did you ever get any resolution or find a way to do what you were trying to do here?  This is a really cool package, but that quirk is bothering me about it.

**Comment by hediet** [comment]
Nope. Luckily I don't need this tool anymore - it was to make symlinks in react native/expo work by just copying the content over symlinking it. I don't use expo anymore.
